crates/invoker-impl/src/invocation_task/mod.rs

```diff
@@ -56,6 +56,10 @@ use tracing::instrument;
 const SERVICE_PROTOCOL_VERSION_V1: HeaderValue =
     HeaderValue::from_static("application/vnd.restate.invocation.v1");
 
+#[allow(clippy::declare_interior_mutable_const)]
+const SERVICE_PROTOCOL_VERSION_V2: HeaderValue =
+    HeaderValue::from_static("application/vnd.restate.invocation.v2");
```

Comment:
We did introduce this new service protocol version to prevent a new SDK from connecting to an old server and assuming that the new retry policy is working but in reality it is not, right?